### PR TITLE
register CoAPS+JPY scheme

### PR DESCRIPTION
**Pull request by mcr**
as per decision at IETF114, register a new scheme for JPY encapsulation

### CODE REVIEW

File: draft-ietf-anima-constrained-join-proxy.md [hunk 1]
```diff
@@ -407,11 +407,13 @@ Upon success, the return payload will contain the join-port of the Registrar.
   REQ: GET coap://[IP_address]/.well-known/core?rt=brski.rjp
 
   RES: 2.05 Content
-  <coaps://[IP_address]:join-port>; rt="brski.rjp"
+  <coaps+jpy://[IP_address]:join-port>; rt="brski.rjp"
 ~~~~
 
 The discoverable port numbers are usually returned for Join Proxy resources in the &lt;URI-Reference&gt; of the payload (see section 5.1 of {{RFC9148}}).
 
+The coaps+jpy scheme is registered is defined in {{jpyscheme}}.
```
Comment (mcr) [inline draft-ietf-anima-constrained-join-proxy.md:415]: Must refer to RFC7252 section 6.2 for details on the scheme.

File: draft-ietf-anima-constrained-join-proxy.md [hunk 2]
```diff
@@ -407,11 +407,13 @@ Upon success, the return payload will contain the join-port of the Registrar.
   REQ: GET coap://[IP_address]/.well-known/core?rt=brski.rjp
 
   RES: 2.05 Content
-  <coaps://[IP_address]:join-port>; rt="brski.rjp"
+  <coaps+jpy://[IP_address]:join-port>; rt="brski.rjp"
```
Comment (EskoDijk) [inline draft-ietf-anima-constrained-join-proxy.md:410]:
```suggestion
  <coaps+jpy://[IP_address]:join-port>;rt=brski.rjp
```

**Review by EskoDijk — APPROVED** [review]
In the IANA registration for the new protocol, why not give it a name? We already use "JPY message" in the draft so we may as well call it "JPY protocol". Then we can indicate that JPY protocol uses the newly defined scheme.

**Comment by mcr** [comment]
> In the IANA registration for the new protocol, why not give it a name? We already use "JPY message" in the draft so we may as well call it "JPY protocol". Then we can indicate that JPY protocol uses the newly defined scheme.

Sure, I think that's fine. We should think for a few minutes if that's a name we really like, but I don't have a better suggestion.

**Review by petervanderstok — APPROVED** [review]
looks OK; pity it was necessary

File: draft-ietf-anima-constrained-join-proxy.md [hunk 3]
```diff
@@ -407,11 +407,13 @@ Upon success, the return payload will contain the join-port of the Registrar.
   REQ: GET coap://[IP_address]/.well-known/core?rt=brski.rjp
 
   RES: 2.05 Content
-  <coaps://[IP_address]:join-port>; rt="brski.rjp"
+  <coaps+jpy://[IP_address]:join-port>;rt=brski.rjp
 ~~~~
 
 The discoverable port numbers are usually returned for Join Proxy resources in the &lt;URI-Reference&gt; of the payload (see section 5.1 of {{RFC9148}}).
 
+The coaps+jpy scheme is registered is defined in {{jpyscheme}}.
```
Comment (mcr) [inline draft-ietf-anima-constrained-join-proxy.md:415]:
```suggestion
The coaps+jpy scheme is registered is defined in {{jpyscheme}}, as per {{RFC7252, Section 6.2}}
```

**Pull request merged**